### PR TITLE
[homework #10] - js-dom by Candelaria Barcelo

### PR DESCRIPTION
**Pull request by candebarcelo**
Hello @VitalikMelnyk , I've finished homework 10 + the optional advanced task.
Thank you in advance for your review.

### CODE REVIEW

File: homeworks/candelaria-barcelo/10-js-dom/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    if (i % 20 === 0) { 
+        newDiv.id = 20 + "/" + Math.ceil(i / 20)
+    } else {
+        newDiv.id = i % 20 + "/" + Math.ceil(i / 20)
+    }
+    document.body.appendChild(newDiv)
+}
+
+// mark when shift key is pressed or not
+shiftKeyStatus = "up"
```
Comment (solvd-mmelnyk) [inline homeworks/candelaria-barcelo/10-js-dom/index.js:14]: it is not good practice to declare variable without `let` or `const`

**Review by solvd-mmelnyk — COMMENTED** [review]
@candebarcelo great work!
please fix small issue and I will merge this PR
also, please make sure that you use prettier for your code

File: homeworks/candelaria-barcelo/10-js-dom/index.js (hunk 1 above)
Comment (candebarcelo) [reply homeworks/candelaria-barcelo/10-js-dom/index.js:14]: ok, corrected in new commit.

**Comment by candebarcelo** [comment]
I've done all the corrections you said. Thanks for all the corrections @solvd-mmelnyk

**Review by VitalikMelnyk — APPROVED** [review]

**Pull request merged**